### PR TITLE
Template merge for nf-core/tools v1.9

### PR DESCRIPTION
**Pull request by christopher-mohr**
Many thanks for contributing to nf-core/epitopeprediction!

Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

## PR checklist
 - [ ] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If necessary, also make a PR on the [nf-core/epitopeprediction branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/epitopeprediction)
 - [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [ ] Make sure your code lints (`nf-core lint .`).
 - [ ] Documentation in `docs` is updated
 - [ ] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/epitopeprediction/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: conf/test.config [hunk 1]
```diff
@@ -8,6 +8,9 @@
  */
 
 params {
+  config_profile_name = 'Test profile'
```
Comment (ggabernet) [inline conf/test.config]: Here in the `test.config`, all the links will be broken, now GitHub only supports `raw.githubusercontant.com`-based links. Could you update them? E.g.
https://raw.githubusercontent.com/nf-core/test-datasets/epitopeprediction/testdata/variants/variants.vcf

Comment (christopher-mohr) [reply conf/test.config]: Thanks @ggabernet. Just to be sure, that affects all (test) config files in `conf/` , right?

Comment (ggabernet) [reply conf/test.config]: yes, that affects them all

Comment (christopher-mohr) [reply conf/test.config]: Looks like it's broken again... :(

Comment (christopher-mohr) [reply conf/test.config]: Ok, I triggered the tests again and now all of them passed. 🎉

**Pull request merged**